dev/archery/setup.py

```diff
@@ -26,7 +26,9 @@
 
 extras = {
     'benchmark': ['pandas'],
-    'bot': ['ruamel.yaml', 'pygithub'],
+    # bot extra includes crossbow's dependencies
```

Comment:
Might it make sense to separate crossbow's dependencies and specify the install as `arrow/dev/archery[bot,crossbow]`?

That way you could install the dependencies with one of the other extras. This is partially because I'm not sure if I should use `arrow/dev/archery[bot]` for what I'm copying in #8915 from https://github.com/apache/arrow/commit/7f11abee01104fceb47088ea42501935a44b0b36#diff-90e4e8bfd9b23a7b4bf4cfa1cb4bd0d80a87e21ac7fbd3a39c30f19709d4c705R50-R61

Reply:
It's a good idea, the same has just crossed my mind during working on #9096. Feel free to add it in #8915.

Reply:
This is repeated a lot, and I'm probably going to just open a different PR for cleaning up the repeated crossbow dependencies that way #8915 can hopefully be merged in sooner rather than later. It looks like it's built correctly (it was working before Travis disappeared).